### ISSUE TITLE
Ports won't terminate after test

### ISSUE THREAD

**Issue by bondjuk** [away3d/away3d-core-openfl#61]
Hello,

On the first test:

```bash
$ cd Basic_View
$ openfl test html5
```
the compiled project runs on Google Chrome. When I close it, the process on bash won't terminate. Therefore I interrupt it with CTRL+Z.

Then I try to test it once more with same commands, I get an EADDRINUSE error on bash which is because node-windows.exe wasn't terminated. So I need to terminate it from bash every time with a command or use taskmgr to find and kill the program.

Is there an easier way to get past this? Could I be doing something wrong about how to terminate a tested project?

**Comment by Greg209** [comment]
This isn't an Away3d specific issue but more related to the test web server use through the OpenFl Test command. Having said that, although I use a Mac I believe that you should be using CTRL+C to terminate the bash script instead of Z. CTRL+Z suspends the process whereas CTRL+C kills the process.

Let me know how you get on.

**Comment by bondjuk** [comment]
Sorry if I posted something irrelevant, but I've never came across such a thing testing any OpenFl examples before. That's why I wanted to ask you guys.

Neither CTRL+Z, nor CTRL+C works for me. How does it terminate on Mac? Does it kill the corresponding application when you close the project on the browser?

**Comment by bondjuk** [comment]
Okay, looks like I figured out the reason:

I tried once on Windows Command Line Prompt and I had no problems. Then I used mintty which I'm running on Cygwin, it couldn't terminate the application again. So that was the reason, looks like the Cygwin bash I'm using doesn't have permissions to kill that application or something else is going wrong while trying to terminate.

Thanks a lot for your attention!
